src/components/IstioWizards/IstioWizard.tsx

```diff
@@ -0,0 +1,265 @@
+import * as React from 'react';
+import { Button, Label, ListView, ListViewIcon, ListViewItem, Wizard } from 'patternfly-react';
+import { WorkloadOverview } from '../../types/ServiceInfo';
+import Slider from './Slider/Slider';
+import { DestinationRule, VirtualService } from '../../types/IstioObjects';
+import { serverConfig } from '../../config';
+import { authentication } from '../../utils/Authentication';
+import * as API from '../../services/Api';
+import * as MessageCenter from '../../utils/MessageCenter';
+
+type Props = {
+  show: boolean;
+  namespace: string;
+  serviceName: string;
+  workloads: WorkloadOverview[];
+  onClose: (changed: boolean) => void;
+};
+
+type WorkloadTraffic = {
+  name: string;
+  traffic: number;
+};
+
+type State = {
+  showWizard: boolean;
+  workloads: WorkloadTraffic[];
+};
+
+class IstioWizard extends React.Component<Props, State> {
+  constructor(props: Props) {
+    super(props);
+    this.state = {
+      showWizard: false,
+      workloads: []
+    };
+  }
+
+  resetState = () => {
+    if (this.props.workloads.length === 0) {
+      return;
+    }
+    const wkTraffic = Math.round(100 / this.props.workloads.length);
+    const remainTraffic = 100 % this.props.workloads.length;
+    const workloads: WorkloadTraffic[] = this.props.workloads.map(workload => ({
+      name: workload.name,
+      traffic: wkTraffic
+    }));
+    if (remainTraffic > 0) {
+      workloads[workloads.length - 1].traffic = workloads[workloads.length - 1].traffic + remainTraffic;
+    }
+    this.setState({
+      showWizard: this.props.show,
+      workloads: workloads
+    });
+  };
+
+  compareWorkloads = (prev: WorkloadOverview[], current: WorkloadOverview[]): boolean => {
+    if (prev.length !== current.length) {
+      return false;
+    }
+    for (let i = 0; i < prev.length; i++) {
+      if (!current.includes(prev[i])) {
+        return false;
+      }
+    }
+    return true;
+  };
```

Comment:
Only proprosal not blocker:
What do you think about ?
```typescript
let found = prev.some( item=> current.includes(item))
return found ? false : true
```

Reply:
I think it wouldn't test the same.
In the code, the idea is that

```
prev:  [A, B, C]
current: [A, B, C, D]

compareWorkloads(prev, current) = false // D is not in prev

but I think that 
prev.some( item => current.includes(item) )
would return true as soon as detect that condition is true for at least one of them but not all.
```

Reply:
ohh I see but you can check not include instead includes